libwget/html_url.c

```diff
@@ -227,6 +243,31 @@ static void _html_get_url(void *context, int flags, const char *tag, const char
 				url.url.len = len;
 				wget_vector_add(res->uris, &url, sizeof(url));
 			}
+
+			char end_char = '"';
+			if (!wget_strcasecmp_ascii(attr, "style")) {
+
+				for (;len && c_isspace(*val); val++, len--); // skip leading spaces
+/*
+				if(len && (*val != '\'') && (*val != '"'))
+					end_char = ' ';							 // https://www.w3.org/TR/html51/syntax.html#elements-attributes
+				else if(len)								 // attribute values can have no quotes at all!
+					end_char = *val;
+
+				val++; len--;
+*/
+
+				char * buf, *temp;
+				if(temp = buf = malloc((len+1) * sizeof(char))) {
+				for(;len && (*val != end_char); len--)
```

Comment:
Feeling embarrassed! Is there a style guide we follow? Sorry I may have missed it.